### ISSUE TITLE
 when I've upgraded this example to spring-boot-starter-parent to 2.0.5. RELEASE ,there is an error in sending authorization requesting address format

### ISSUE THREAD

**Issue by zhengzizhi** [kbastani/spring-cloud-event-sourcing-example#26]
First of all, thank you very much for sharing the example, 
and I've benefited a lot from your example, 
which I've upgraded spring-boot-starter-parent to 2.0.5. RELEASE and 
upgraded spring-security-oauth2 to 2.3.3.RELEASE

When I clicked the http://192.168.99.100:8787 home page login link, the requested address was incorrect (return code = 404):
Http://192.168.99.100:8181/oauth/authorize?Client_id=acme&redirect_uri=http://192.168.100:8787/login&response_type=code&state=nIXDJD

I hope correct address should be like this:
Http://192.168.99.100:8181/uaa/oauth/authorize?Client_id=acme&redirect_uri=http://192.168.99.100:8787/login&response_type=code&state=nIXDJD

I don't know how to solve this problem.

My Example address:
https://github.com/zhengzizhi/microservice-store

**Comment by zhengzizhi** [comment]
如何在CentOS 7上配置Java的Maven开发环境
https://blog.csdn.net/zhengzizhi/article/details/81039627

**Comment by ghost** [comment]
@kbastani - Will you please update this project to Spring Boot Version 2.1.0.RELEASE?

**Comment by kbastani** [comment]
@JavaHelper Yes, I am working on upgrading Spring Boot across all my reference examples at the moment. I'll have an update soon.

**Comment by ghost** [comment]
@kbastani  - Please let me know once project is upgraded. I'm anxiously waiting for it.

**Comment by ghost** [comment]
@kbastani - Did you get chance to upgrade the project ?
